torch/nn/functional.pyi.in

```diff
@@ -166,7 +166,9 @@ def log_softmax(input: Tensor, dim: Optional[int] = ..., _stacklevel: int = ...,
 def tanh(input: Any): ...
 
 
-def sigmoid(input: Any): ...
+def sigmoid(input: Any) -> Tensor: ...
+
+def hardsigmoid(input: Tensor, inplace: bool = False) -> Tensor: ...
```

Comment:
if we are changing them here, should we also add typing on sigmoid, hardsigmoid, silu in functional.py too?

Reply:
@walterddr those functions are already annotated (and exist in functional.py)